metaflow/plugins/argo/argo_client.py

```diff
@@ -57,6 +57,24 @@ def get_workflow_template(self, name):
                 json.loads(e.body)["message"] if e.body is not None else e.reason
             )
 
+    def get_workflow(self, name):
+        try:
+            return self._client.CustomObjectsApi().get_namespaced_custom_object(
+                group=self._group,
+                version=self._version,
+                namespace=self._namespace,
+                plural="workflows",
+                name=name,
+            )
+        except self._client.rest.ApiException as e:
+            if e.status == 404:
+                print("404 error - workflow not found")
+                return None
+            print("non-404 error")
```

Comment:
Suggest removing these prints to keep stdout clean.

Function that calls this class should check for `None` return value.

Reply:
done